content/src/content/jcr_root/apps/core/wcm/components/cloudviewer/v1/cloudviewer/cloudviewer.html

```diff
@@ -0,0 +1,53 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2020 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<sly data-sly-use.cloudViewerModel="com.adobe.cq.wcm.core.components.models.CloudViewer"/>
+
+<style>
+     .cloudviewer:not(.cq-Editable-dom) .adobe-dc-view-full-window-borderless {
+          height: 100vh;
+          position: fixed;
+          width: 100vw;
+          top: 0;
+          left: 0;
+          z-index: 9999;
+     }
+
+     .cloudviewer.cq-Editable-dom .adobe-dc-view-full-window-borderless {
+          height: 500px;
+     }
+
+     .adobe-dc-view-full-window, .adobe-dc-view-sized-container {
+          height: ${cloudViewerModel.viewerHeight @ context='styleToken'} !important;
+     }
+</style>
+
+<h2 data-sly-test="${!cloudViewerModel.documentPath}">Please select a document to display.</h2>
+
+<div id="adobe-dc-view" class="adobe-dc-view ${cloudViewerModel.containerClass @ context='html'}"></div>
+<script src="https://documentcloud.adobe.com/view-sdk/main.js"></script>
```

Comment:
See comment at line#18, CSS and JS should only be added once to the page. You can refactor them to be generic and package them as AEM clientlibs. For vendor dependencies (DC view SDK) you can add them to the page header conditionally (when a DC viewer config is available or, even better, if the component is actually used on the page).

Reply:
Fixed in 57dcdf2